src/App/App.js

```diff
@@ -28,8 +28,8 @@ var App = React.createClass({
     if (!selectedStory["selected"]) {
       selectedStories++;
       selectedStory["selected"] = true;
-      this.setState({selectedStory:selectedStory, selectedStories:selectedStories});
     }
+    this.setState({selectedStory:selectedStory, selectedStories:selectedStories});
```

Comment:
This should mean that the selectedStory changes each time, even if the number of previously-selected-stories doesn't change.